config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Should this file be deleted once you've switched the load_defaults in application.rb? That's what I've been doing...